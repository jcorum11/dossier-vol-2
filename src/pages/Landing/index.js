import { Fragment } from 'react'
import Jumbotron from '../../components/Jumbotron'
const Landing = () => {
  return (
    <Fragment>
      <Jumbotron></Jumbotron>
    </Fragment>
  )
}

export default Landing
