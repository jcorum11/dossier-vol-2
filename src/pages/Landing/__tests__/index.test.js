import { render, cleanup } from '@testing-library/react'
import Landing from '..'

afterEach(cleanup)

it('renders', async () => {
  render(<Landing />)
})
it('Matches Snapshot', async () => {
  const { asFragment } = render(<Landing />)
  expect(asFragment()).toMatchSnapshot()
})