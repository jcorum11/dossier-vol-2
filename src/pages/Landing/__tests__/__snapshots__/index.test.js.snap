// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL fmtvOV"
  >
    <div
      class="sc-gsDKAQ jogOtt"
    />
  </div>
</DocumentFragment>
`;
